src/main/java/lulu/TaskList.java

```diff
@@ -0,0 +1,92 @@
+package lulu;
+
+import lulu.task.Deadline;
+import lulu.task.Event;
+import lulu.task.Task;
+import lulu.task.Todo;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> list;
+
+    public TaskList() {
+        this.list = new ArrayList<>();
+    }
+
+    public void add(Task t) {
+        this.list.add(t);
+    }
+
+    public void remove(int taskNumber) {
+        list.remove(taskNumber - 1);
+    }
+
+    public int getSize() {
+        return this.list.size();
+    }
+
+    public Task get(int i) {
+        return this.list.get(i);
+    }
+
+    public String getRecentTaskDescription() {
+        return this.list.get(list.size() - 1).toString();
+    }
+
+    public String getTaskDescription(int taskNumber) {
+        return this.list.get(taskNumber - 1).toString();
```

Comment:
Perhaps use a separate method for getting the correct index to make it clear why `taskNumber - 1` needs to be done?